itou/templates/apply/submit/application/resume.html

```diff
@@ -202,7 +202,7 @@
                         <p class="font-weight-bold fs-sm mb-1">Vous n’avez pas de CV ?</p>
                         <p class="fs-sm mb-2">Créez-en un grâce à notre partenaire Diagoriente.</p>
                     {% endif %}
-                    <a href="https://plateforme.diagoriente.beta.gouv.fr"
+                    <a href="https://diagoriente.beta.gouv.fr/services/plateforme?utm_source=emploi-inclusion-{% if request.user.is_prescriber %}prescripteur{% else %}candidat{% endif %}"
```

Comment:
```suggestion
                    <a href="https://diagoriente.beta.gouv.fr/services/plateforme?utm_source=emploi-inclusion-{{ request.user.is_prescriber|yesno:"prescripteur,candidat" }}"
```
Je me dit que ça allège peut-être un peu :shrug: